control_plane/scripts/tests/test_uninstall.py

```diff
@@ -0,0 +1,520 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache-2 License.
+
+# This product includes software developed at Datadog (https://www.datadoghq.com/) Copyright 2025 Datadog, Inc.
+
+# stdlib
+import json
+import sys
+from pathlib import Path
+from unittest import TestCase
+from unittest.mock import patch as mock_patch
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
```

Comment:
if i'm reading this right, this is allowing 15 to work. might be good to have a comment making the reason for this path addition more explicit.